src/item.h

```diff
@@ -133,23 +130,42 @@ struct iteminfo {
         /** Whether to print sName.  If false, use for comparisons but don't print for user. */
         bool bDrawName;
 
+        /** Whether to print a sign on positive values */
+        bool bShowPlus;
+
+        enum flags {
+            no_flags = 0,
+            is_decimal = 1 << 0, ///< Print as decimal rather than integer
+            no_newline = 1 << 1, ///< Do not follow with a newline
+            lower_is_better = 1 << 2, ///< Lower values are better for this stat
+            no_name = 1 << 3, ///< Do not print the name
+            show_plus = 1 << 4, ///< Use a + sign for positive values
+        };
+
         /**
          *  @param Type The item type of the item this iteminfo belongs to.
          *  @param Name The name of the property this iteminfo describes.
+         *  @param flags Additional flags to customize this entry
          *  @param Fmt Formatting text desired between item name and value
          *  @param Value Numerical value of this property, -999 for none.
-         *  @param _is_int If true then Value is interpreted as an integer
          *  @param Plus Character to place before value, generally '+' or '$'
```

Comment:
This parameter is also gone (or was never there).

Reply:
Thanks, fixed.